tests/test_repository.py

```diff
@@ -183,7 +183,11 @@ def testFlat(self):
         self.assertEqual(flat.getDetector().getId(), self.ccdNum)
 
     def testLinearizer(self):
-        self.assertTrue(self.butler.get('linearizer', ccdnum=1))
+        lin1 = self.butler.get('linearizer', ccdnum=1)
+        lin2 = self.butler.get('linearizer', ccdnum=2)
+        self.assertTrue(lin1)
+        self.assertTrue(lin2)
```

Comment:
I'm not sure these `assertTrue`s mean much — what does it mean for a `Linearizer` to be `True`?

If you just want to check that you have an object of some sort, I'd suggest `assertIsNotNone`.

Reply:
Aren't you wanting to test that the instances are distinct? (so checking that `id(lin1) != id(line2)` ?

Reply:
that's the next line...

Reply:
I can change it to `assertIsNotNone`. The original code in this test (before my modifications) had `assertTrue`.